### ISSUE TITLE
Execute tests under other supported sysemts

### ISSUE THREAD

**Issue by Kielek** [open-telemetry/opentelemetry-dotnet-instrumentation#2006]
Execute e2e tests under other supported systems (mainly linux).  See https://github.com/dotnet/core/blob/main/release-notes/6.0/supported-os.md#linux.

Tests could be executed only after merge to main and create issues if it fails.

**Comment by pjanotti** [comment]
@Kielek it seems a duplicate of #715, if you agree, please, close the older one as a duplicate.

**Comment by Kielek** [comment]
@pjanotti, thanks it was on my todo list to check with you.

**Comment by Kielek** [comment]
Closing. I think that current coverage is cool enough.
